pkg/dynatrace/dynatrace.go

```diff
@@ -151,6 +151,31 @@ func (dynatraceApiClient *DynatraceApiClient) MakeRequest(method, path string, r
 	return dynatraceApiClient.httpClient.Do(req)
 }
 
+func (dynatraceApiClient *DynatraceApiClient) GetListDynatraceHttpMonitors(clusterId string) (ExistsHttpMonitorInDynatraceResponse, error) {
```

Comment:
This might be more obvious if named `GetHttpMonitorsForCluster`. Which also calls out that the functions in this file are a bit redundantely named. The package and type having `dynatrace` in the name means it would be simpler to have `dynatraceClient.GetThing()` versus `dynatrace.GetDynatraceThing()` but no need to fix that in this PR.

Reply:
Agree, that is a good point WRT the naming. 
Resolving comment without fix as almost every function has the same issue and I want to keep this PR scoped right now